plugins/catalog/src/components/EntityContextMenu/EntityContextMenu.tsx

```diff
@@ -144,6 +154,16 @@ export function EntityContextMenu(props: EntityContextMenuProps) {
             </ListItemIcon>
             <ListItemText primary="Inspect entity" />
           </MenuItem>
+          <MenuItem
+            onClick={() => {
+              copyToClipboard();
```

Comment:
you should call onClose() here too so that the popup closes after clicking the entry